v2/index.html

```diff
@@ -1578,6 +1578,37 @@ <h5>Tied Notes</h5>
                 <h5>Grace Notes</h5>
                 <div class="issue" data-number="22"></div>
                 <div class="issue" data-number="75"></div>
+                <p>
+                    There are two types of grace note: Acciaccatura and appogiatura.
+                </p>
+                <p>
+                    For a single acciaccatura the lower-case character <code>g</code>
+                    MUST be used. This character MUST be specified in the order given above;
+                    that is, it MUST precede the note name, and MUST also precede all other
+                    optional attributes of the note. Consecutive single acciaccatura
+                    SHOULD NOT occur.
+                </p>
+                <p>
+                    There are two methods used to encode appogiatura.
+                </p>
+                <p>
+                    For a single appogiatura, the lower-case character <code>q</code>
+                    MUST be used. This character MUST be specified in the order given above;
+                    that is, it MUST precede the note name, and MUST also precede all other
+                    optional attributes of the note. Consecutive single appogiatura
+                    MAY occur.
```

Comment:
```suggestion
                    optional attributes of the note. Consecutive single appogiatura
                    MAY occur even though they SHOULD be coded with an appogiatura group.
```

Reply:
Updated it -- "SHOULD" implied "MAY", so the first part was redundant.

Reply:
It might be technically the same but being explicit draws the attention on the existence of appogiatura groups. I think it would be important to have a sentence recommending to use `qq...r` instead of `q.q.q.`.

Reply:
Maybe something for the encoding guidelines?

Reply:
Not sure. The point is that there are not the same and that you can express more things with a `qq...r`. I would rather remove the sentence, or even forbid them and fix our data.

Reply:
What's the difference between these two except that one is using single notes, and the other not? 

`A{q8Bq8''C}{'AB}/4A{qq8B''Cr}{'AB}`

<img width="242" alt="image" src="https://github.com/rism-digital/pae-code-spec/assets/163183/ed104476-9ad1-439a-b95b-30fc257a4d19">

I thought I would compare the MEI, but that only raised more questions than answers.
 
<details>

<summary>MEI Output</summary>

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/5.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/5.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="5.0">
   <meiHead>
      <fileDesc>
         <titleStmt>
            <title />
         </titleStmt>
         <pubStmt>
            <date isodate="2024-05-27-13:54:26" />
         </pubStmt>
      </fileDesc>
      <encodingDesc>
         <appInfo>
            <application xml:id="verovio" version="4.2.0-8a772f5">
               <name>Verovio (4.2.0-8a772f5)</name>
            </application>
         </appInfo>
         <projectDesc>
            <p>MEI encoded with Verovio</p>
            <p>Converted from Plaine and Easie to MEI</p>
         </projectDesc>
      </encodingDesc>
   </meiHead>
   <music>
      <body>
         <mdiv xml:id="m15wwqth">
            <score xml:id="sv3lqdm">
               <scoreDef xml:id="s2gqwr3">
                  <staffGrp xml:id="s8j2ung">
                     <staffDef xml:id="s1ahluu1" n="1" lines="5" clef.shape="G" clef.line="2" />
                  </staffGrp>
               </scoreDef>
               <section xml:id="s1pfogn4">
                  <measure xml:id="mfnjo7e" right="single">
                     <staff xml:id="s1mj6nxn" n="1">
                        <layer xml:id="l1d0qcvg" n="1">
                           <note xml:id="nmwwo00" dur="4" oct="4" pname="a" />
                           <beam xml:id="b4ks4fv">
                              <note xml:id="n1s4ffnc" dur="8" oct="4" pname="b" grace="acc" stem.dir="up" />
                              <note xml:id="n9ixwzy" dur="8" oct="5" pname="c" grace="acc" stem.dir="up" />
                           </beam>
                           <beam xml:id="b147azdb">
                              <note xml:id="n1ffvwuh" dur="8" oct="4" pname="a" />
                              <note xml:id="nzosgrv" dur="8" oct="4" pname="b" />
                           </beam>
                        </layer>
                     </staff>
                  </measure>
                  <measure xml:id="mhbsl03" right="invis">
                     <staff xml:id="seeh8fz" n="1">
                        <layer xml:id="l5odh2d" n="1">
                           <note xml:id="n1q84ocf" dur="4" oct="4" pname="a" />
                           <beam xml:id="b1kzq82m">
                              <graceGrp xml:id="ga734np">
                                 <note xml:id="n1gzuhvg" dur="8" oct="4" pname="b" />
                                 <note xml:id="n1vw1hrc" dur="8" oct="5" pname="c" />
                              </graceGrp>
                           </beam>
                           <beam xml:id="bshell2">
                              <note xml:id="n36inxr" dur="8" oct="4" pname="a" />
                              <note xml:id="na56g6u" dur="8" oct="4" pname="b" />
                           </beam>
                        </layer>
                     </staff>
                  </measure>
               </section>
            </score>
         </mdiv>
      </body>
   </music>
</mei>
```

</details>

 - There is no indication in the `<graceGrp>` of what kind of grace note it is. I'm guessing `<graceGroup>` should also provide for `@grace`?
 - Or do the notes inside of `<graceGrp>` need to be marked with `@grace`? How do you know the `<graceGroup>` is an appogiatura?
 - `@ornam` is also available? When would you use `@ornam="A"` vs `@grace="acc"`? (Also it's confusing that "Appogiatura" is `acc`, since you might otherwise think that it is "acciaccatura" unless you know that it's actually "Accented")

Still, it's not clear to me how they are different, other than they are encoded differently, and that one should be preferred over the other.